src/howler.core.js

```diff
@@ -338,6 +359,36 @@
     },
 
     /**
+     * Obtain an activated HTML5 Audio object from the pool. If there are no
+     * pooled objects left (or mobileAutoEnable is false), a new Audio
+     * object without activation will be returned.
+     * @return {Audio}
+     */
+    _obtainHtml5Audio: function() {
+      var self = this || Howler;
+      var audio;
+
+      if (self._mobileAudioPool && self._mobileAudioPool.length) {
+        audio = self._mobileAudioPool.pop();
+      }
+
+      return audio || new Audio();
```

Comment:
Correct. This is a fallback to ensure audio still has a _chance_ of working if the pool is over scheduled. It's probably worth a warning in the console however.